### PR TITLE
Adding Dockerfile

### PR DESCRIPTION
**Pull request by vaijab**
This allows us to build a docker image.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,10 @@
+FROM vaijab/nodejs:0.12.5
+
+WORKDIR /brp_app
+
+COPY package.json /brp_app/package.json
+RUN npm install
+COPY . /brp_app
+
+EXPOSE 8080
+CMD npm start
```
Comment (JoeChapman) [inline Dockerfile]: what about `npm install` here?

**Comment by JoeChapman** [comment]
@vaijab said "it kinda makes sense for people to merge their own PRs once others reviewed ...
because PR owner knows best when to integrate the PR"

I would rather stick to the reviewer merging for now and have an open discussion about it next weeke with @easternbloc included.

File: Dockerfile (hunk 1 above)
Comment (vaijab) [inline Dockerfile]: Yeah, so the reason I add `package.json` first is to benefit from `docker build` caching.

**Comment by vaijab** [comment]
> I would rather stick to the reviewer merging for now and have an open discussion about it

Sounds good to me. So feel free to merge it then :-)

**Pull request merged**